site/topics/searching/searching.rst

```diff
@@ -96,15 +96,15 @@ Binary Search
 .. admonition:: Activity
     :class: activity
 
-    I will guess a number between :math:`0 -- 1023` in :math:`10` or less guesses.
+    I will guess a number between :math:`0` -- :math:`1023` in :math:`10` or less guesses.
 
     There is a catch however --- you have to tell me if the number is *higher* or *lower* than my guess before I guess
     again.
 
 
 * This is an example of a *binary search*
 * Like linear search, binary search is used to find an element within some collection
-* However, if I was doing a linear search for a number between :math:`0 -- 1023`, I could only guarantee that I would find the element in :math:`1024` guesses
+* However, if I was doing a linear search for a number between :math:`0` -- :math:`1023`, I could only guarantee that I would find the element in :math:`1024` guesses
```

Comment:
Another typical way to write this is `[0..1023]` which is a little cleaner to typeset.